### DISCUSSION TITLE
How to get individual vertical scrollbars for splitter?

### DISCUSSION THREAD

**Discussion by JS-Aibel** [zauberzeug/nicegui#806] [Q&A] [answered]
When using the [splitter](https://nicegui.io/documentation/splitter), 
how do I get an independent vertical scrollbar for each section like shown in the [Quasar's Splitter component](https://quasar.dev/vue-components/splitter#basic)?

**Comment by falkoschindler** [comment, marked as answer]
You do get independent scrollbars:

```py
with ui.splitter().classes('w-[200px] h-[300px]') as splitter:
    with splitter.before:
        for i in range(100):
            ui.label(i)
    with splitter.after:
        for i in range(100):
            ui.label(hex(i))
```

It's important though to restrict the splitter size. Otherwise it just grows with its content and there is no need for scrollbars.

**Reply by JS-Aibel** [reply]
Thank you, I was missing the splitter size restriction.
